src/components/VerifyVoterInfo.jsx

```diff
@@ -42,17 +42,11 @@ class VerifyVoterInfo extends Component {
     );
   }
 
-  _renderVoterInfo = (el) => {
-    return (
-      <Button name="voterAddressGroup" key={el.voter_reg_num} onClick={() => this._handleAddressClick(el)}>{el.resident_address}</Button>
-    );
-  }
+  // eslint-disable-next-line
+  _renderVoterInfo = (el) => { return (<Button name="voterAddressGroup" key={el.voter_reg_num} onClick={() => this._handleAddressClick(el)}>{el.resident_address}</Button>); }
 
-  _renderCandidates = (candidate) => {
-    return (
-      <p>{candidate}</p>
-    );
-  }
+  // eslint-disable-next-line
+  _renderCandidates = (candidate) => { return ( <p>{candidate}</p> ); }
```

Comment:
Returning JSX with arrow functions can be done in the same way I mentioned above.
```
_renderCandidates = (candidate) => ( <p>{candidate}</p> );
```